webapp/channels/src/components/admin_console/message_export_settings.jsx

```diff
@@ -31,7 +31,8 @@ export default class MessageExportSettings extends AdminSettings {
                 SMTPUsername: this.state.globalRelaySMTPUsername,
                 SMTPPassword: this.state.globalRelaySMTPPassword,
                 EmailAddress: this.state.globalRelayEmailAddress,
-
+                CustomSMTPServerName: this.state.globalRelayCustomSMTPServerName,
+                CustomSMTPPort: this.state.globalRelayCustomSMTPPort,
             };
```

Comment:
Should the values for ServerName and Port be configurable via the system console?

Reply:
This is just a quick test for a prospect, we will add that at a later date if our change works for them.

Reply:
1/5 that if don't add the UI setting right now, we won't in the future. Is it a substantial amount of work to do that as part of this PR?

Reply:
Done, it was small